addons/sale/views/sale_portal_templates.xml

```diff
@@ -573,7 +574,7 @@
                                                 <t t-out="((line.discount % 1) and '%s' or '%d') % line.discount"/>%
                                             </strong>
                                         </td>
-                                        <td t-attf-class="text-end {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}" id="taxes">
+                                        <td t-attf-class="text-end {{ 'd-none' if not show_invoice_tax else '' }} {{ 'd-none d-md-table-cell' if report_type == 'html' and show_invoice_tax else '' }}" id="taxes">
```

Comment:
Why not a simple `t-if` on the `td` ?
Is there a reason to add invisible content to the report ?

Reply:
Fixed!